Makefile

```diff
@@ -290,11 +290,8 @@ JUNIT_REPORT ?= /dev/null
 bench:
 	@go test -count 1 -bench=. -benchmem -run=^$$ ./internal/...
 
-.PHONY: test
-test: test.unit
```

Comment:
How about not dropping this target but adding `test.envtest` to it? Both unit and envtests are fast enough to be run locally very often and IMO it would make sense to keep this target as a first-to-go local check.

Reply:
Might be misleading without any suffix: it could indicate that all tests would be run. WDYT?

Reply:
Maybe `test.fast` then? 🤔 (or anything similar?)

Reply:
And turning `test.all` into just `test`?

Reply:
I'm not sure about this. I'd be more inclined into leaving this as is and just running `make test.unit test.envtest` if anyone wants to perform a "fast" test round.